network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -198,18 +198,22 @@ <h2 class="h3-heading">{% trans "Security" %}</h2>
           {% blocktrans with url="https://foundation.mozilla.org/privacynotincluded/about/meets-minimum-security-standards" asvar minimum_security_standards trimmed %}
             Does this product meet our <a href="{{ url }}">Minimum Security Standards</a>?
           {% endblocktrans %}
-          {% trans "Encryption"              as encryption %}
-          {% trans "Security updates"        as security_updates %}
-          {% trans "Strong password"         as strong_password %}
-          {% trans "Manages vulnerabilities" as manages_vulnerabilities %}
-          {% trans "Privacy policy"          as privacy_policy %}
-
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.meets_minimum_security_standards|yes_no  help=product.uses_encryption_helptext         label=minimum_security_standards|safe  no_border=True    ding=product.show_ding_for_minimum_security_standards %}
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.uses_encryption                          help=product.uses_encryption_helptext         label=encryption                  no_border=True %}
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.strong_password                          help=product.strong_password_helptext         label=strong_password             no_border=True %}
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.security_updates                         help=product.security_updates_helptext        label=security_updates            no_border=True %}
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.manage_vulnerabilities                   help=product.manage_vulnerabilities_helptext  label=manages_vulnerabilities     no_border=True %}
-          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.privacy_policy                           help=product.privacy_policy_helptext          label=privacy_policy              no_border=True %}
+          {% include "fragments/product_criterion.html"  with  show_value_as_symbol=False  value=product.meets_minimum_security_standards|yes_no  help=product.uses_encryption_helptext         label=minimum_security_standards|safe  no_border=True    ding_control=product.show_ding_for_minimum_security_standards   class="meets-minimum-security-standards" %}
+          <input type="checkbox" id="mss-accordion-toggle">
+          <label for="mss-accordion-toggle"><span class="sr-only">Toggle expandable section</span></label>
```

Comment:
Let’s translate this string please :)
I know it’s easy to miss since it’s not visible, but people using screen readers also deserve translated content

```suggestion
          <label for="mss-accordion-toggle"><span class="sr-only">{% trans "Toggle expandable section" context "Accessibility label for an accordion toggle" %}</span></label>
```

Reply:
good catch